frontends/precinct-scanner/src/screens/scan_error_screen.tsx

```diff
@@ -19,13 +19,15 @@ export function ScanErrorScreen({ error, isTestMode }: Props): JSX.Element {
     switch (error) {
       // Invalid interpretation
       case 'invalid_test_mode':
-        return isTestMode ? 'Live Ballot detected.' : 'Test ballot detected.';
+        return isTestMode
+          ? 'Live Ballot detected. Scanner is in test mode.'
+          : 'Test ballot detected. Scanner is in live mode.';
       case 'invalid_election_hash':
-        return 'Scanned ballot does not match the election this scanner is configured for.';
+        return 'The ballot does not match the election this scanner is configured for.';
       case 'invalid_precinct':
-        return 'Scanned ballot does not match the precinct this scanner is configured for.';
+        return 'The ballot does not match the precinct this scanner is configured for.';
       case 'unreadable':
-        return 'There was a problem reading this ballot. Please try again.';
+        return 'There was a problem reading this ballot. Please scan again.';
       case 'unknown':
```

Comment:
Based on the latest implementation, I believe this specific `unknown` state is non-recoverable by removing the ballot, so we may want to prompt a poll worker to restart the tablet. No need to change this yet, until we finalize the implementation as we have a bunch of pending fixes that may make this point moot.

Reply:
So, this error occurs when either front or back is interpreted as 'UninterpretedHmpbPage'. I don't know what that actually means.

In any case, VxScan _will_ recover when you remove the ballot in this case.

Reply:
So should I update this state to add a returned message "Please remove your ballot."?

<img width="1313" alt="image" src="https://user-images.githubusercontent.com/28444/183498441-3df091a2-c2dd-4dc5-85b8-081f049db3b4.png">

Reply:
Perhaps no action needed here if we update the last sentence per Roe's suggestion: https://github.com/votingworks/vxsuite/pull/2276/files#r938938765

Reply:
I would like to know why this happens. @benadida do you have any idea what 'UninterpretedHmpbPage' means? As far as I can tell from diving into the code, it's whenever there's an error interpreting a HMPB, but I'm not sure what those errors might be.

I'm tempted to just treat this the same as "unreadable" and ask them to try again. I'm sure they'll eventually give up if it doesn't work a few times.

Reply:
That's right, it is things like can't find the expected contest boxes, either not right shape or number. Could be because of some marginal marks confusing things.

This is also likely the error that we'll use if we detect too weird a skew.

This should result at least in having the voter try again once or twice.

Reply:
Thanks. Updating `unknown` and `unreadable` to display the same message:

> There was a problem reading this ballot. Please scan again.